src/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixer.php

```diff
@@ -66,75 +66,73 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+            $docContent = $tokens[$docIndex]->getContent();
 
-                // ignore one-line phpdocs like `/** foo */`, as there is no place to put new annotations
-                if (false === strpos($docContent, "\n")) {
-                    continue;
-                }
+            // ignore one-line phpdocs like `/** foo */`, as there is no place to put new annotations
+            if (false === strpos($docContent, "\n")) {
+                return;
+            }
 
-                $doc = new DocBlock($docContent);
+            $doc = new DocBlock($docContent);
 
-                // skip if already has annotation
-                if (!empty($doc->getAnnotationsOfType([
-                    'covers',
-                    'coversDefaultClass',
-                    'coversNothing',
-                ]))) {
-                    continue;
+            // skip if already has annotation
+            if (!empty($doc->getAnnotationsOfType([
+                'covers',
+                'coversDefaultClass',
+                'coversNothing',
+            ]))) {
+                return;
+            }
+        } else {
+            $docIndex = $index;
+            $tokens->insertAt($docIndex, [
+                new Token([T_DOC_COMMENT, sprintf('/**%s%s */', $this->whitespacesConfig->getLineEnding(), $indent)]),
+                new Token([T_WHITESPACE, sprintf('%s%s', $this->whitespacesConfig->getLineEnding(), $indent)]),
+            ]);
+
+            if (!$tokens[$docIndex - 1]->isGivenKind(T_WHITESPACE)) {
+                $extraNewLines = $this->whitespacesConfig->getLineEnding();
+
+                if (!$tokens[$docIndex - 1]->isGivenKind(T_OPEN_TAG)) {
+                    $extraNewLines .= $this->whitespacesConfig->getLineEnding();
                 }
-            } else {
-                $docIndex = $index;
+
                 $tokens->insertAt($docIndex, [
-                    new Token([T_DOC_COMMENT, sprintf('/**%s%s */', $this->whitespacesConfig->getLineEnding(), $indent)]),
-                    new Token([T_WHITESPACE, sprintf('%s%s', $this->whitespacesConfig->getLineEnding(), $indent)]),
+                    new Token([T_WHITESPACE, $extraNewLines.$indent]),
                 ]);
-
-                if (!$tokens[$docIndex - 1]->isGivenKind(T_WHITESPACE)) {
-                    $extraNewLines = $this->whitespacesConfig->getLineEnding();
-
-                    if (!$tokens[$docIndex - 1]->isGivenKind(T_OPEN_TAG)) {
-                        $extraNewLines .= $this->whitespacesConfig->getLineEnding();
-                    }
-
-                    $tokens->insertAt($docIndex, [
-                        new Token([T_WHITESPACE, $extraNewLines.$indent]),
-                    ]);
-                    ++$docIndex;
-                }
-
-                $doc = new DocBlock($tokens[$docIndex]->getContent());
+                ++$docIndex;
             }
 
+            $doc = new DocBlock($tokens[$docIndex]->getContent());
+        }
```

Comment:
By having moved this close-brace up, the remainder of the method is indented wrongly.

This means that either `braces` isn't doing its job correctly, or you didn't run `php-cs-fixer fix`, and none of the PR-style-checks are picking it up 😕

Can you run `php-cs-fixer fix` and see if it indents correctly?